Script.js

```diff
@@ -1,3 +1,118 @@
+    img: 'images/project-cart-snapshoot.svg',
+    demoLink: 'https://ab-noori.github.io/My-Portfolio',
+    sourceLink: 'https://github.com/ab-noori/My-Portfolio',
+  },
+  {
+    id: '5',
+    title: 'Multi-Post Stories Gain+Glory',
+    desc: 'Lorem, ipsum dolor sit amet consectetur adipisicing elit. Voluptas alias omnis non consequuntur. Labore blanditiis adipisci facilis tenetur minima, dolores voluptatem earum deserunt, laudantium harum incidunt.',
+    technogies: ['Ruby on rails', 'CSS', 'HTML', 'JavScript'],
+    img: 'images/project-cart-snapshoot.svg',
+    demoLink: 'https://ab-noori.github.io/My-Portfolio',
+    sourceLink: 'https://github.com/ab-noori/My-Portfolio',
+  },
+  {
+    id: '6',
+    title: 'Multi-Post Stories Gain+Glory',
+    desc: 'Lorem, ipsum dolor sit amet consectetur adipisicing elit. Voluptas alias omnis non consequuntur. Labore blanditiis adipisci facilis tenetur minima, dolores voluptatem earum deserunt, laudantium harum incidunt.',
+    technogies: ['Ruby on rails', 'CSS', 'HTML', 'JavScript'],
+    img: 'images/project-cart-snapshoot.svg',
+    demoLink: 'https://ab-noori.github.io/My-Portfolio',
+    sourceLink: 'https://github.com/ab-noori/My-Portfolio',
+  },
+];
+
+function generatProjectCarts() {
+  projectCarts.innerHTML = cartsData.map((x) => {
+    const {
+      id, title, desc, technogies, demoLink, sourceLink,
+    } = x;
+    return `
+          <div class="recent-work-carts">
+              <div class="cart-snapshoot">
+                
+              </div>
+              <div class="cart-summary">
+                  <h2 class="project-title">${title}</h2>
+                  <ul class="project-tags">
+                      <li >${technogies[0]}</li>
+                      <li >${technogies[1]}</li>
+                      <li >${technogies[2]}</li>
+                      <li >${technogies[3]}</li>
+                  </ul>
+                  <button class="btn"  data-modal-target="#modal${id}">See Project</button>
+              </div>
+          </div>
+
+          <div class="modal" id="modal${id}">
+            <button data-modal-close-button class="modal-header-close-button">&times;</button>
+            <div class="popup-snapshoot">
+              <img src="images/desktop-popup-image.svg" width="100%" id="popup-snapshoot-desktop" alt="desktop popup snapshoot">
+              <img src="images/mobile-popup-image.svg" id="popup-snapshoot-mobile" alt="mobile popup snapshoot">
+            </div>
+            <div class="modal-header">
+              <div class="modal-header-content">
+                <div class="modal-header-title">${title}</div>
+                <div class="modal-top-links">
+                  <button class="btn"><a href="${demoLink}"> See Live <img src="images/Icons/Icon-see-live.svg" alt="Link icon"></a></button>
+                  <button class="btn"><a href="${sourceLink}"> See Source <img src="images/Icons/Icon-see-source.svg" alt="GitHub icon"></a></button>
```

Comment:
- [Optional] Good job making the changes @ab-noori, One thing I would suggest is to use less font sizing for your links and try to make it more aligned with the figma design. 

### Figma design
![217996592-93dad669-d3ac-40e5-92b0-67a39a4ce412](https://user-images.githubusercontent.com/69077061/218025230-e58c7a28-5457-40cc-9b17-93bdedfd40e2.png)

### Your design 
![Screenshot 2023-02-10 100446](https://user-images.githubusercontent.com/69077061/218025240-8c412c1e-4ef0-41f1-8555-599c0d61d418.png)